tst/testinstall/cyclotom.tst

```diff
@@ -276,6 +276,8 @@ he integer 0)
 gap> SetCyclotomicsLimit(100);
 Error, SetCyclotomicsLimit: <newlimit> must not be less than old limit of 1000\
 000
+gap> SetCyclotomicsLimit(2^32);
+Error, Cyclotomic field size limit must be less than 2^32
```

Comment:
Thanks. Unfortunately, this now fails on 32bit systems. To solve that, you can use `#@if 8*GAPInfo.BytesPerVariable = 64`... See for example `tst/test32bit/2017-06-20-restrictedperm.tst`  (sorry, I should have thought about that and warned you)